### PR TITLE
SPUI-1263 > Deposit UI fixes

### PR DESCRIPTION
**Pull request by VovaMedvediev**

### CODE REVIEW

File: lib/app/shared/features/crypto_deposit/view/crypto_deposit.dart [hunk 1]
```diff
@@ -13,7 +13,7 @@ import 'components/crypto_deposit_with_address_and_tag/crypto_deposit_with_addre
 import 'components/deposit_info.dart';
 import 'components/show_deposit_disclaimer.dart';
 
-class CryptoDeposit extends HookWidget {
+class CryptoDeposit extends StatefulHookWidget {
```
Comment (eli1stark) [inline lib/app/shared/features/crypto_deposit/view/crypto_deposit.dart:16]: use HookWidget instead

File: lib/app/shared/features/crypto_deposit/view/crypto_deposit.dart [hunk 2]
```diff
@@ -23,39 +23,55 @@ class CryptoDeposit extends HookWidget {
...
     final colors = useProvider(sColorPod);
-    useProvider(cryptoDepositDisclaimerFpod(currency.symbol).select((_) {}));
-    final deposit = useProvider(cryptoDepositNotipod(currency));
-    final depositN = useProvider(cryptoDepositNotipod(currency).notifier);
+    useProvider(
+      cryptoDepositDisclaimerFpod(widget.currency.symbol).select((_) {}),
+    );
+    final deposit = useProvider(cryptoDepositNotipod(widget.currency));
+    final depositN =
+        useProvider(cryptoDepositNotipod(widget.currency).notifier);
```
Comment (eli1stark) [inline lib/app/shared/features/crypto_deposit/view/crypto_deposit.dart:48]: add `final controller = useScrollController();`
useScrollController hook will dispose scollController automatically

File: lib/app/shared/features/crypto_deposit/view/crypto_deposit.dart [hunk 3]
```diff
@@ -23,39 +23,55 @@ class CryptoDeposit extends HookWidget {
...
 
+  @override
+  _CryptoDepositState createState() => _CryptoDepositState();
+}
+
+class _CryptoDepositState extends State<CryptoDeposit> {
+  late ScrollController _scrollController;
+
+  @override
+  void initState() {
```
Comment (eli1stark) [inline lib/app/shared/features/crypto_deposit/view/crypto_deposit.dart:34]: remove

File: lib/app/shared/features/crypto_deposit/view/crypto_deposit.dart [hunk 4]
```diff
@@ -23,39 +23,55 @@ class CryptoDeposit extends HookWidget {
   final String header;
   final CurrencyModel currency;
 
+  @override
+  _CryptoDepositState createState() => _CryptoDepositState();
+}
+
+class _CryptoDepositState extends State<CryptoDeposit> {
+  late ScrollController _scrollController;
```
Comment (eli1stark) [inline lib/app/shared/features/crypto_deposit/view/crypto_deposit.dart:31]: remove

**Review by eli1stark — CHANGES_REQUESTED** [review]

File: plugins/simple_kit/lib/src/shared/components/qr_code_fields/simple_address_field_with_copy.dart [hunk 5]
```diff
@@ -10,6 +10,7 @@ class SAddressFieldWithCopy extends HookWidget {
     Key? key,
     this.realValue,
     this.onTap,
+    this.scrollToFullCopyMessage,
```
[the remark this answers is not in the document]
Comment (VovaMedvediev) [reply plugins/simple_kit/lib/src/shared/components/qr_code_fields/simple_address_field_with_copy.dart:13]: done

File: plugins/simple_kit/lib/src/notifications/view/simple_notification_box.dart [hunk 6]
```diff
@@ -22,28 +22,32 @@ class SNotificationBox extends StatelessWidget {
...
+          padding: const EdgeInsets.symmetric(
+            horizontal: 20.0,
+          ),
           child: Row(
             crossAxisAlignment: CrossAxisAlignment.start,
             children: [
-              SErrorIcon(
-                color: SColorsLight().white,
+              Padding(
+                padding: const EdgeInsets.symmetric(vertical: 20),
```
[the remark this answers is not in the document]
Comment (VovaMedvediev) [reply plugins/simple_kit/lib/src/notifications/view/simple_notification_box.dart:32]: done

File: lib/app/shared/features/crypto_deposit/view/crypto_deposit.dart (hunk 1 above)
Comment (VovaMedvediev) [reply lib/app/shared/features/crypto_deposit/view/crypto_deposit.dart:16]: done

File: lib/app/shared/features/crypto_deposit/view/crypto_deposit.dart (hunk 2 above)
Comment (VovaMedvediev) [reply lib/app/shared/features/crypto_deposit/view/crypto_deposit.dart:48]: done
cool, didn't know about this

File: lib/app/shared/features/crypto_deposit/view/crypto_deposit.dart (hunk 3 above)
Comment (VovaMedvediev) [reply lib/app/shared/features/crypto_deposit/view/crypto_deposit.dart:34]: done

File: lib/app/shared/features/crypto_deposit/view/crypto_deposit.dart (hunk 4 above)
Comment (VovaMedvediev) [reply lib/app/shared/features/crypto_deposit/view/crypto_deposit.dart:31]: done

**Pull request merged**